### ISSUE TITLE
IzzyOnDroid repo only offers x86_64 build

### ISSUE THREAD

**Issue by TubaApollo** [lotusgate/otraku#85]
Hey, first of all thank you for this great app. It seems like that the F-Droid (IzzyOnDroid) repo only has the x86_64 build, therefore it doesnt work on many devices. Is this a mistake, or any other reason? Would be great to have arm64 build on there as well :)

**Comment by lotusgate** [comment]
Yeah, before I only uploaded the full apk, this is the first time I split the ABIs and it picked the wrong one. Next time I will also provide the universal apk, so it should work (we will figure out what's wrong, if it picks the wrong file again).

**Comment by zoroya1** [comment]
> Yeah, before I only uploaded the full apk; this is the first time I split the ABIs and Izzy picked the wrong one. Next time I will also provide the universal apk, so it should work (we will figure out what's wrong, if it picks the wrong file again).

Can you point out the difference between play store version and Izzy version?
I installed the play store version since Izzy's version failed to update
